### ISSUE TITLE
[enhancement] Way to expose require to browser console

### ISSUE THREAD

**Issue by natew** [steelbrain/pundle#150]
Something I've wanted over time to have. Basically would be nice to have a way to `require()` any of your node_modules from browser console. Could be a plugin for pundle.

Bonus feature: allow `require('./my/app/file.js')`.

I tried using __sbPundle.require and requireModule, but both threw errors for me.

**Comment by steelbrain** [comment]
What are the benefits of allowing this?

**Comment by natew** [comment]
Lots of times when you are debugging something, you want access to the library. Without this, you need to go into your code, add a `window.xyz = require()`, and then hard refresh (because you had a runtime error), and then wait for load and debug.

With this, you can just go into console and `require()` and debug.

One example I ran into recently was a infinite loop caused by passing an object to the library `kontur` twice. It would have saved me a bit of time if I could have done this.

**Comment by steelbrain** [comment]
I think `window.xyz = require()` and then HMR should be good enough.

To support this we'll have to majorly revamp the whole transformation system, as when you do `require()` without an absolute path, it would have to emulate the lookup algo in browser without access to a file-system. It also gets complicated with the recent file-types since you wouldn't be able to specify format in require.

tl;dr: when compiled `require('./something')` turns into `require('js_491237451')` in the browser. There's no way to resolve a `require('kontur')` in the browser. You'll have to do it through code and then HMR
